packages/rum/test/viewCollection.spec.ts

```diff
@@ -170,7 +172,81 @@ describe('rum track loading time', () => {
     jasmine.clock().tick(AFTER_PAGE_ACTIVITY_END_DELAY)
     jasmine.clock().tick(THROTTLE_VIEW_UPDATE_PERIOD)
 
-    expect(getViewEvent(1).loadingTime).toEqual(BEFORE_PAGE_ACTIVITY_VALIDATION_DELAY)
+    expect(getViewEvent(1).loadingTime).toEqual(msToNs(BEFORE_PAGE_ACTIVITY_VALIDATION_DELAY))
+  })
+
+  const FAKE_NAVIGATION_ENTRY = {
+    domComplete: 456,
+    domContentLoadedEventEnd: 345,
+    domInteractive: 234,
+    entryType: 'navigation',
+    loadEventEnd: 567,
+  }
+  const FAKE_NAVIGATION_ENTRY_2 = {
+    domComplete: 2,
+    domContentLoadedEventEnd: 1,
+    domInteractive: 1,
+    entryType: 'navigation',
+    loadEventEnd: 1,
+  }
+
+  it('should use loadEventEnd for initial view when having no activity', () => {
```

Comment:
can you keep these constants inside the corresponding tests or move them with the other constants at the top of the file?